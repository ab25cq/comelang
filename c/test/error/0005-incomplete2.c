/*
PATTERN: incomplete
PATTERN: c:10:
*/

struct X x;

int main()
{
	return x.v;
}

struct X {int v;};
