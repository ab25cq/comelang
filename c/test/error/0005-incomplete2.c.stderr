selector on incomplete type at test/error/0005-incomplete2.c:10:13
    return x.v;
            ^
